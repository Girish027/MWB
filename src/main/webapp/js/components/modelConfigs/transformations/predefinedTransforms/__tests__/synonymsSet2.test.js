import synonymsSet2 from 'components/modelConfigs/transformations/predefinedTransforms/synonymsSet2';

describe('synonymsSet2', () => {
  const regex = synonymsSet2();

  test('type is equal to regex-replace', () => {
    expect(regex.type).toEqual('regex-replace');
  });

  test('mapping contains more than one key', () => {
    const keys = Object.keys(regex.mappings);
    expect(keys.length).toBeGreaterThan(1);
  });

  test('snapshot predefined transformation', () => {
    expect(regex.mappings).toMatchSnapshot();
  });
});
