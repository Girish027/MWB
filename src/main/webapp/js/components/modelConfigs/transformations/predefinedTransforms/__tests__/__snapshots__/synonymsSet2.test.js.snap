// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`synonymsSet2 snapshot predefined transformation 1`] = `
Object {
  "/\\\\b( front.?desk\\\\S*|frontdesk\\\\S*|k.?osk\\\\S*|k.?.?osk\\\\S*|reception\\\\S*|booth\\\\S*|counter|counters )\\\\b/i": "FrontDesk",
  "/\\\\b(( ( turn\\\\S*|shut\\\\S*) && ( off\\\\S*|down ) )|disabl\\\\S*|disconnect\\\\S*|dis.?connect\\\\S*|unconnect\\\\S*|un.?connect\\\\S*|unplug\\\\S*|shutdown\\\\S*|shut\\\\S* down\\\\S*|shut\\\\S* off\\\\S* )\\\\b/i": "Disconnect",
  "/\\\\b((Apple && Pay\\\\S*)|Appl.?Pay\\\\S*|Apple.?Pay\\\\S* )\\\\b/i": "ApplePay",
  "/\\\\b((credit\\\\S* && (rating\\\\S*|histor\\\\S*|bureau\\\\S*|b.?rrow\\\\S*|report\\\\S*|scor\\\\S*))|equ.?fax\\\\S*|mogo)\\\\b/i": "CreditRating",
  "/\\\\b((financ\\\\S* && (software\\\\S*|soft.?ware|program\\\\S*))|quicken|quickbooks|turbotax|turbo.?tax)\\\\b/i": "FinancialSoftware",
  "/\\\\b(America\\\\S*|united states|united states of america|usa|u.s.a.|us|u.s.|states|the states|murica\\\\S*|.?murica\\\\S*)\\\\b/i": "America",
  "/\\\\b(Amount\\\\S*|quantit\\\\S*|volume\\\\S*|length\\\\S*|weight\\\\S*|total\\\\S*|size\\\\S*)\\\\b/i": "Amount",
  "/\\\\b(Foreign\\\\S* Exchang\\\\S*|Forex|For.?Ex|FX\\\\S*|AFX\\\\S*|((exchang\\\\S*|conver\\\\S*) && rate\\\\S*))\\\\b/i": "ForeignExchange",
  "/\\\\b(acces\\\\S*|acess\\\\S*|retr.?.?v\\\\S*|(g.?t\\\\S* && (into|onto|in|on|in.?to|on.?to))|enter\\\\S*|entry\\\\S* )\\\\b/i": "Access",
  "/\\\\b(account\\\\S*|acount\\\\S*|accont\\\\S*|accoutn\\\\S*|acoutn\\\\S*|a.?c|accnt\\\\S*|acnt\\\\S*|acct)\\\\b/i": "Account",
  "/\\\\b(acquir\\\\S*|obtain\\\\S*|get|getting|geting|gets|got|rec.?.?v\\\\S*|earn\\\\S*|grab\\\\S*|procur\\\\S*|ach.?.?v\\\\S*|attain\\\\S*)\\\\b/i": "Acquire",
  "/\\\\b(admit\\\\S*|admmit\\\\S*|addmit\\\\S*|admis\\\\S*|admmis\\\\S*|addmis\\\\S*|g.?t\\\\S* in)\\\\b/i": "Admit",
  "/\\\\b(adult\\\\S*|mature\\\\S*|grown.?up\\\\S*|((over|more.?th.?n|older.?th.?n) && (18|eighteen|21|twenty.?one|twentyone)))\\\\b/i": "Adult ",
  "/\\\\b(advant\\\\S*|ben.?fit\\\\S*|gain\\\\S*|plus\\\\S*|pros|perk\\\\S*|priv.?l\\\\S*|privl\\\\S*)\\\\b/i": "Advantage",
  "/\\\\b(after|afterward\\\\S*|following|subsequent\\\\S*|behind\\\\S*|later|succeeding|below\\\\S*|ensuing|next )\\\\b/i": "After ",
  "/\\\\b(agent\\\\S*|expert\\\\S*|advis.?r\\\\S*|rep|reps|representative\\\\S*|csr\\\\S*|cxr\\\\S*)\\\\b/i": "Agent",
  "/\\\\b(agree\\\\S*|acquies\\\\S*|assent\\\\S*|comply\\\\S*|conced\\\\S*|concur\\\\S*|consent\\\\S*)\\\\b/i": "Agree",
  "/\\\\b(airport\\\\S*|airpl\\\\S*|plane\\\\S*|terminal\\\\S*|air\\\\S* craft\\\\S*|aircraft\\\\S*|flight\\\\S*)\\\\b/i": "AirTravel",
  "/\\\\b(alc.?h.?l\\\\S*|beer\\\\S*|booz\\\\S*|liqu.?r\\\\S*|cocktail\\\\S*)\\\\b/i": "Alcohol",
  "/\\\\b(alumn\\\\S*|alum|((class of|graduat\\\\S* in ) && (19.?.?|20.?.?)) )\\\\b/i": "Alumni",
  "/\\\\b(antivirus|anti.?virus|sophos|mcaf\\\\S*|norton\\\\S*|symant.?c|trend microsystem\\\\S*|kapersky\\\\S*|kapersk\\\\S* lab\\\\S*|Malwarebyte\\\\S*|mal.?ware.?byte\\\\S*| ADW Cleaner|Rogue Killer|Junkware Removal tool\\\\S*|JRT|CCleaner|CC cleaner|c cleaner)\\\\b/i": "Antivirus",
  "/\\\\b(apply\\\\S*|applic\\\\S*|aply\\\\S*|appy|aplic\\\\S* )\\\\b/i": "Apply",
  "/\\\\b(appointment\\\\S*|apointment\\\\S*|appt\\\\S*|meet\\\\S*|see|visit\\\\S*|check.?up\\\\S*|checkup\\\\S* )\\\\b/i": "Appointment",
  "/\\\\b(assign\\\\S*|del.?gate\\\\S*|designat\\\\S*|appoint\\\\S*|nominat\\\\S*|tag\\\\S* )\\\\b/i": "Assign",
  "/\\\\b(attend\\\\S*|atend\\\\S*|presenc\\\\S*|participat\\\\S*|((go|goes|going|gone|went) && (to|somewhere))|show\\\\S* up )\\\\b/i": "Attend",
  "/\\\\b(avail\\\\S*|vacan\\\\S*|open\\\\S*|on hand|on.?hand|unoccu\\\\S*|unocu\\\\S*|access.?bl\\\\S*|acces.?bl\\\\S*|handy\\\\S*|ready\\\\S*)\\\\b/i": "Available",
  "/\\\\b(averag\\\\S*|medi.?n\\\\S*|avg )\\\\b/i": "Average",
  "/\\\\b(bag\\\\S*|box|boxes|cargo|luggage\\\\S*|lugguage\\\\S*|laugage\\\\S*|lougage\\\\S*|suitcase\\\\S*|suit.?case\\\\S*|backpack\\\\S*|back.?pack\\\\S*|duffle.?bag\\\\S*|case\\\\S*|trunk\\\\S*|carry\\\\S* on|carry.?on|check.?in lug\\\\S*|check\\\\S* in lug\\\\S*|check.?in bag\\\\S*)\\\\b/i": "Baggage",
  "/\\\\b(bal.?nc\\\\S*|bal|owe|owes|owed|owing )\\\\b/i": "Balance",
  "/\\\\b(bank\\\\S* branch\\\\S*|branch\\\\S*|financial\\\\S* center\\\\S*|financial\\\\S* centre\\\\S*|teller\\\\S* )\\\\b/i": "BankBranch",
  "/\\\\b(bill\\\\S*|invoic\\\\S*|iou\\\\S*|i.?o.?u\\\\S*|william)\\\\b/i": "Bill",
  "/\\\\b(chang\\\\S*|lower\\\\S*|decre.?s\\\\S*|drop\\\\S*|go\\\\S* down\\\\S*|reduc\\\\S*|diminish\\\\S*|shrink\\\\S*|abat\\\\S*|low|less|small\\\\S*|minim\\\\S*)\\\\b/i": "Decrease",
  "/\\\\b(deadline\\\\S*|dead.?line\\\\S*|due|due.?date\\\\S*|((final\\\\S*|last\\\\S*|latest) && (hour\\\\S*|day\\\\S*|week\\\\S*|month\\\\S*|year\\\\S*|date\\\\S*|time\\\\S* )))\\\\b/i": "Deadline",
  "/\\\\b(debit\\\\S* card\\\\S*|debit.?card\\\\S*|debitcard\\\\S*|debit\\\\S*|bank\\\\S* card\\\\S*|bankcard\\\\S*|access.?card\\\\S*|accesscard\\\\S*|acess.?card\\\\S*|acesscard\\\\S*|ATM card\\\\S*|ABM card\\\\S*|cash machine\\\\S* card\\\\S*|bank machine\\\\S* card\\\\S*|conv\\\\S* card\\\\S*|client card\\\\S*|client.?card\\\\S*)\\\\b/i": "DebitCard",
  "/\\\\b(deductible\\\\S*|dedutible\\\\S*|deduc\\\\S*|pay\\\\S* out of pocket|policy payment\\\\S*)\\\\b/i": "Deductible",
  "/\\\\b(defer\\\\S*|put\\\\S*off\\\\S*|delay\\\\S*|layover\\\\S*|lay\\\\S* over\\\\S*|prolong\\\\S*|protract\\\\S*|postpon\\\\S*|post.?pon\\\\S* )\\\\b/i": "Defer",
  "/\\\\b(deposit\\\\S*|contrib\\\\S*|put\\\\S* in|add\\\\S* to )\\\\b/i": "Deposit",
  "/\\\\b(educate\\\\S*|education\\\\S*|teach|teaches|teaching|taught|tutor|tutors|tutoring|train|trains|trained|training|instruct|instruction\\\\S*|edu|ed|learn\\\\S*|schooling|studies|tutelag\\\\S*|learn\\\\S* )\\\\b/i": "Education",
  "/\\\\b(eligib\\\\S*|criteri\\\\S*|qualif\\\\S* )\\\\b/i": "Eligibility ",
  "/\\\\b(email\\\\S*|e.?mail\\\\S*|gmail|outlook|hotmail|yahoo\\\\S*)\\\\b/i": "Email",
  "/\\\\b(employ|employment|employs|employed|work\\\\S*|job\\\\S*|career\\\\S*|becom\\\\S* an employ\\\\S*|work\\\\S* for|work\\\\S* with|work\\\\S* at|recruit\\\\S*|hir\\\\S*|CV|C.V.|( ( position\\\\S*|opening\\\\S* ) && avail\\\\S* )|placement\\\\S*|resume\\\\S*|hr|hum.?n\\\\S* resourc\\\\S*|position\\\\S*)\\\\b/i": "Employment",
  "/\\\\b(exam|exams|exam.?s|exam.?nation\\\\S*|mid.?term\\\\S*|midterm\\\\S*|test|tests|test.?s|testing|final|finals|quiz|asses\\\\S*|asess\\\\S*)\\\\b/i": "ExamTest",
  "/\\\\b(exchang\\\\S*|trad\\\\S* in|trade\\\\S*|trading|barter\\\\S*|swap\\\\S*|bunz )\\\\b/i": "Exchange",
  "/\\\\b(exempt\\\\S*|absolv\\\\S*|except\\\\S*|free.?of|immun\\\\S*|let.?off|(not && (liab\\\\S*|responsib\\\\S*|subject\\\\S*))|void.?of|writ\\\\S* off)\\\\b/i": "Exempt",
  "/\\\\b(expir\\\\S*|conclus\\\\S*|culminat\\\\S*|finish\\\\S*|done|run\\\\S* out|perish\\\\S*)\\\\b/i": "Expire",
  "/\\\\b(fax\\\\S*|facs\\\\S*)\\\\b/i": "Fax",
  "/\\\\b(fee|fees|dues|pen.?lt\\\\S*|pen.?li\\\\S*|fine\\\\S*|retribution\\\\S*|toll|tolls )\\\\b/i": "Fee",
  "/\\\\b(financing|finance\\\\S*|fundrais\\\\S*|debtleverag\\\\S*|debt.?leverag\\\\S*)\\\\b/i": "Financing",
  "/\\\\b(fix\\\\S*|repair\\\\S*|correct\\\\S*|adjust\\\\S*|.?mend\\\\S*|patch\\\\S*|restor\\\\S*|mend\\\\S*|revamp\\\\S*|tune\\\\S*)\\\\b/i": "Fix",
  "/\\\\b(foreclos\\\\S*|fore.?clos\\\\S*)\\\\b/i": "Foreclosure",
  "/\\\\b(fraud\\\\S*|fruad|scam\\\\S*|ponzi|scam artist\\\\S*|phish\\\\S*|suspicious\\\\S*|extort\\\\S*|swindl\\\\S*|embezzl\\\\S*|counterf\\\\S*|fake\\\\S*|false|falsified|ingenuine\\\\S*|phony|bogus|imitat\\\\S*|spurious|ersatz|feign\\\\S*|sham|forge\\\\S*|phonies|falsify)\\\\b/i": "Fraud",
  "/\\\\b(free|no.?charge|compl.?ment.?r\\\\S*|on.?the.?house|no.?cost|gratis|no.?fee\\\\S*|without charge\\\\S*|without fee|without fees|toll.?free)\\\\b/i": "Free",
  "/\\\\b(function\\\\S*|work\\\\S*|operat\\\\S*|perform\\\\S*|behav\\\\S*|produc\\\\S* result\\\\S*|respond\\\\S*)\\\\b/i": "Function",
  "/\\\\b(gym|gyms|g.?mnasium\\\\S*|gym.?s|fitnes\\\\S*|work\\\\S* out\\\\S*|workout\\\\S*|work.?out\\\\S*|exer.?i.?e\\\\S*|( (fitn.?s\\\\S*|recreation\\\\S*|rec ) && ( center\\\\S*|centre\\\\S*|complex\\\\S*|building\\\\S*|facil\\\\S* ) ))\\\\b/i": "Fitness",
  "/\\\\b(hotel\\\\S*|resort\\\\S*|motel\\\\S*|inn\\\\S*|accoma\\\\S*|accomi\\\\S*|accomo\\\\S*|accommo\\\\S*|acoma\\\\S*|acomi\\\\S*|acomo\\\\S*|lodg\\\\S*|villa|villas|chalet\\\\S*|chatea\\\\S*|bed\\\\S* breakfast\\\\S*|bed\\\\S* .? breakfast\\\\S*|hostel\\\\S*|airbnb\\\\S*|air.?bnb\\\\S*|bnb|B .? B)\\\\b/i": "Hotel",
  "/\\\\b(hours|((hour\\\\S*|time\\\\S*) && (busines\\\\S*|buisnes\\\\S*|biz|bus.?|operat\\\\S*|offic\\\\S*|open\\\\S*|clos\\\\S*)))\\\\b/i": "Hours",
  "/\\\\b(illness|ill|sick\\\\S*|nause\\\\S*|heartburn|indigestion|upset stomach|diarrhea\\\\S*|diarrhoea\\\\S*|under.?the.?weather|flu|flue|influenz\\\\S*|( ( stomach|tummy|belly) && ach\\\\S* )|migrain\\\\S*|vomit\\\\S*|unwell|ailing|ailment\\\\S*|diseas\\\\S*|a.?cold|cough\\\\S*|malaria\\\\S*|tetan.?s\\\\S*|tetn.?s\\\\S*|chickenpox|chicken.?pox|shingles|infect\\\\S*|ear.?infect\\\\S*|fever\\\\S*|allerg\\\\S*|measle\\\\S*|mumps\\\\S*|rubella\\\\S*|pneumonia\\\\S*|neumonia\\\\S*|diabet\\\\S* )\\\\b/i": "Illness ",
  "/\\\\b(imm.?gr.?nt\\\\S*|im.?gr.?nt\\\\S*|imm.?gr.?t\\\\S*|im.?gr.?t\\\\S*)\\\\b/i": "Immigrant",
  "/\\\\b(immediat\\\\S*|imediat\\\\S*|now|instant\\\\S*|promp\\\\S*|straightaway\\\\S*|straight.?away\\\\S*|right.?away\\\\S*|pressing\\\\S*|urgent\\\\S* )\\\\b/i": "Immediately ",
  "/\\\\b(infant\\\\S*|baby\\\\S*|babe|babies|toddler\\\\S*|new.?born\\\\S*|newborn\\\\S*|just.?born)\\\\b/i": "Infant ",
  "/\\\\b(info\\\\S*|data\\\\S*|datum|detail\\\\S*|descrip\\\\S*|describ\\\\S*|defin\\\\S* )\\\\b/i": "Information",
  "/\\\\b(insur.?n.?e\\\\S*|plan|plans|policy\\\\S*|policies\\\\S*|cover\\\\S*|health\\\\S* ben.?f\\\\S*|ben.?fic\\\\S*|( ( health|med\\\\S*|group\\\\S*|insur\\\\S* ) && ( plan|plans|scheme\\\\S*|arrangement\\\\S*|program\\\\S*|ben.?f.?t\\\\S*|package\\\\S*|cover\\\\S*|polic\\\\S* ) ))\\\\b/i": "InsurancePolicy",
  "/\\\\b(insur\\\\S*|cover\\\\S*|protect\\\\S* )\\\\b/i": "Insure",
  "/\\\\b(inter.?st rate\\\\S*|rate\\\\S* of inter.?st|rate\\\\S*|APR|A.P.R\\\\S*)\\\\b/i": "InterestRate",
  "/\\\\b(inter.?st|intr.?st)\\\\b/i": "Interest",
  "/\\\\b(limit\\\\S*|cap|capping\\\\S*|caps\\\\S*|capped\\\\S*|bound\\\\S*|ceiling\\\\S*|floor\\\\S*|cut.?off\\\\S*|cutoff\\\\S*|cut\\\\S* off\\\\S*|allow.?nce\\\\S*|alow.?nce\\\\S* )\\\\b/i": "Limit",
  "/\\\\b(link\\\\S*|join\\\\S*|add|adds|added|adding|attach\\\\S*|attatch\\\\S*|coupl\\\\S*|relat\\\\S*|conjoin\\\\S*)\\\\b/i": "Link",
  "/\\\\b(mail\\\\S*|snailmail\\\\S*|snail.?mail\\\\S*|post|postage|maling )\\\\b/i": "Mail",
  "/\\\\b(maintain\\\\S*|keep\\\\S*|kept|save\\\\S*|saving\\\\S*|retain\\\\S*|continue\\\\S*|sustain\\\\S*|uph.?ld\\\\S*|upkeep\\\\S*|up.?keep\\\\S*|care\\\\S* for|caring for )\\\\b/i": "Maintenance",
  "/\\\\b(many|multipl\\\\S*|numerous\\\\S*|several\\\\S*|more|extra|extras|supplement\\\\S*|suplement\\\\S*|addition\\\\S*|adition\\\\S*|another|lots|(a && (lot|few|couple))|alot|more  th.?n|one more|more th.?n one|2|3|4|5|6|7|8|9|10|two|three|four|five|six|seven|eight|nine|ten|2nd|3rd|4th|5th|6th|7th|8th|9th|10th|second|third|fourth|forth|fifth|sixth|seventh|eighth|ninth|tenth)\\\\b/i": "Many",
  "/\\\\b(mar.?juan\\\\S*|pot|weed|420|toke|toking|bong|splif\\\\S*|hash|thc|cann.?bis\\\\S*|can.?bis\\\\S*|ganja\\\\S*|stoner|stoned|g.?t.?high)\\\\b/i": "Marijuana",
  "/\\\\b(med\\\\S* practi\\\\S*|doctor\\\\S*|PCP|P.C.P.|physician\\\\S*|primary care physician\\\\S*|dr|DR.?|DR.?s|physician\\\\S*|family doctor\\\\S*|family practitioner\\\\S*|general practitioner\\\\S*|family physician\\\\S*|nurse|nurses|nursing|gen\\\\S* practi\\\\S*|GP|GPs|GP.?s|G.P. )\\\\b/i": "MedicalPractitioners",
  "/\\\\b(medication\\\\S*|pr.?scri\\\\S*|meds|perscri.?\\\\S*|drug\\\\S*|pharm\\\\S*|med.?cin\\\\S*|script|scripts|pill|pills )\\\\b/i": "Medication ",
  "/\\\\b(messag\\\\S*|((text\\\\S*|txt) && (messag\\\\S*|msg\\\\S*))|text\\\\S*|txt\\\\S*|msg\\\\S*|sms\\\\S*|mms\\\\S*|bbm\\\\S*|imessag\\\\S*|i.?messag\\\\S*|whatsapp\\\\S*|whats.?app\\\\S*|what.?s.?app\\\\S*|slack)\\\\b/i": "Messaging ",
  "/\\\\b(meter\\\\S*|metre\\\\S*|mter\\\\S*|metter\\\\S*|meater\\\\S*|meeter\\\\S*|gauge\\\\S*|guage\\\\S*)\\\\b/i": "Meter",
  "/\\\\b(microsoft suite|microsoft\\\\S*|((ms|microsoft|micro.?soft) && (office|word|excel|exel|ecxel|ecsel|ecel|onedrive|one.?drive|powerpoint\\\\S*|power.?point\\\\S*|ppt\\\\S*|onenote|one.?note|outlook))|spread sheet\\\\S*|spread.?sheet\\\\S*|excel.?|exel.?|ecxel.?|ecsel.?|ecel.?|word|msword|word  process\\\\S*|office|onedrive|one.?drive|powerpoint\\\\S*|power.?point\\\\S*|ppt\\\\S*|slideshow\\\\S*|slide.?show\\\\S*|onenote|one.?note|outlook)\\\\b/i": "MicrosoftSuite",
  "/\\\\b(min|minim\\\\S*|least)\\\\b/i": "Minimum",
  "/\\\\b(mobile.?app\\\\S*|phone.?app\\\\S*|app|apps )\\\\b/i": "MobileApp",
  "/\\\\b(money\\\\S*|monies|doll.?r\\\\S*|dol.?r\\\\S*|fund\\\\S*|cash\\\\S*|curenc\\\\S*|currenc\\\\S*|wealth\\\\S*|legal tender|dough|dolla billz|bucks|buck|coin|coins )\\\\b/i": "Money",
  "/\\\\b(municipal\\\\S*|city\\\\S*|cities|town\\\\S*|villag\\\\S*|metro\\\\S*|urban\\\\S*|suburb\\\\S*|county\\\\S*|counties)\\\\b/i": "Municipality",
  "/\\\\b(mutu.?l fund\\\\S*|money.?market\\\\S*|MF|bond fund\\\\S*|invest\\\\S* fund\\\\S*|close\\\\S* end fund\\\\S*|stock\\\\S* fund\\\\S*|hedge fund\\\\S* )\\\\b/i": "MutualFund",
  "/\\\\b(name\\\\S*|naming|firstname\\\\S*|first.?name\\\\S*|lastname\\\\S*|last.?name\\\\S*|surname\\\\S* )\\\\b/i": "Name",
  "/\\\\b(no|nothing\\\\S*|none\\\\S*|nada\\\\S*|zero|not|aren.?t|arent|daren.?t|darent|didn.?t|didnt|doesn.?t|doesnt|does not|don.?t|dont|did not|hasn.?t|hasnt|haven.?t|havent|hadn.?t|hadnt|is not|isn.?t|isnt|mayn.?t|maynt|mightn.?t|mightnt|mustn.?t|mustnt|needn.?t|neednt|oughtn.?t|oughtnt|shan.?t|shant|shouldn.?t|shouldnt|wasn.?t|wasnt|weren.?t|werent|won.?t|wont|wouldn.?t|wouldnt|will not|neither|nor|never)\\\\b/i": "Negative",
  "/\\\\b(number\\\\S*|no.|numer\\\\S*|digit|digits)\\\\b/i": "Number",
  "/\\\\b(online\\\\S*|on.?line\\\\S*|internet\\\\S*|internt\\\\S*|net|web\\\\S*|virtual\\\\S* )\\\\b/i": "Online",
  "/\\\\b(timeframe\\\\S*|time.?frame\\\\S*|how long does it take|how many days|timeline|duration\\\\S*)\\\\b/i": "Timeframe",
  "/\\\\b(transfer\\\\S*|trasfer\\\\S*|move\\\\S*|moving|relocat\\\\S*|re.?locat\\\\S*)\\\\b/i": "Transfer",
  "/\\\\b(warr.?nty|warr.?nties|war.?nty|war.?nties|garant\\\\S*|g.?.?rant\\\\S*|protect\\\\S* plan\\\\S*|protectionplan\\\\S*|protection.?plan\\\\S*)\\\\b/i": "Warranty",
  "/\\\\b(youth\\\\S*|child\\\\S*|kid\\\\S*|teen\\\\S*|youngster\\\\S*|boy\\\\S*|girl\\\\S* )\\\\b/i": "Youth",
}
`;
